site/public/js/calendar.js

```diff
@@ -408,12 +592,9 @@ function generateFullCalendar(start, end, semester_name) {
     // Header area: two buttons to move, and month
     const table = generateCalendarHeader(buildSemesterHeader(semester_name));
     const tableBody = document.createElement('tbody');
-
```

Comment:
All of these whitespace changes are unnecessary and should be reverted due to the risk of merge conflicts, given that multiple PRs are touching similar code.  Apply throughout.

Reply:
Oh, yea those were accidental, I appreciate you catching them. Is there any way to just easily remove them?